k8s/cassandra/chart/cassandra/templates/application.yaml

```diff
@@ -11,8 +11,8 @@ metadata:
     app.kubernetes.io/name: "{{ .Release.Name }}"
 spec:
   descriptor:
-    type: Cassandra
-    version: '3.11'
+    type: Cassandra (Managed Updates)
```

Comment:
On master we don't want to have the " (Managed Updates)" suffix in application.yaml

Reply:
This diff was due to wrong base. Outdated